### ISSUE TITLE
[DatePicker] - Narrator is staying mute on invoking next/previous month buttons.

### ISSUE THREAD

**Issue by Bhavana-Nimma** [microsoft/fluentui#18996]
### Environment Information

- **Package version(s)**: 8.21.1
- **Browser and OS versions**: Internet Explorer (IE11), Microsoft Edge legacy, Windows 10

### Describe the issue:

On invoking next/previous month buttons in the callout, narrator is staying mute and not announcing anything.

#### Actual behavior:

In the below example, on invoking previous month button from "June", month got changed to "May" but narrator stays mute.

![image](https://user-images.githubusercontent.com/85894656/126143366-9ee0811b-c331-482b-b64e-d25d601b435c.png)

#### Expected behavior:

On invoking the previous month button from "June", Narrator should announce as "May 2021" (newly opened month) followed by focus element i.e "Go to previous month April, button".

![image](https://user-images.githubusercontent.com/85894656/126145949-7a97516f-2875-4ab3-ac13-fc031ed982b8.png)

**Comment by ling1726** [comment]
@Bhavana-Nimma I cannot repro the same behaviour you describe, could you share a code repro of what you are experiencing ?

![image](https://user-images.githubusercontent.com/20744592/126155129-e641256e-89c6-4ade-9eea-5b4089e7b37f.png)

**Comment by Bhavana-Nimma** [comment]
@ling1726 Can you please tell me the browser in which you are trying?

This is the issue in IE11 and Edge Legacy. We are using fluent version 8* but the home page (`https://developer.microsoft.com/en-us/fluentui`) and the examples page of fluent version 8* (`https://fabricweb.z5.web.core.windows.net/oufr/8.21.1/`)  are not loading in IE11 whereas the page loads with fluent version 7* (`https://fabricweb.z5.web.core.windows.net/oufr/7.159.0/`). So I am unable to give you the exact code repro of fluent 8* component in IE11. 

We are able to load our feature pages which uses fluent 8* components and it is when we are facing this narrator issue.

**Comment by ling1726** [comment]
Ah I see, I didn't read the browser field correctly, I was using chromium edge. I'm not entirely sure how to run legacy edge. With the august EOL for IE11 in M365, that browser might not be that important anymore

@smhigley  @JustSlone what is the v8 SLA for IE11 and legacy Edge ?

**Comment by smhigley** [comment]
There are two things going on here:
1. Live regions in IE11, especially with Narrator, are notoriously unlikely to work.
2. I'm not convinced we should have a live region there at all, for any screen reader. It's effectively telling the user "yes, you successfully clicked that button you just clicked", especially since the button name includes the month it's navigating to.

Since this issue isn't creating a bad Narrator experience with IE11, I'm not sure it's worth spending time on. It certainly wouldn't be worth changing the live region experience for other more common screen reader/browser pairs. We generally accept that IE11 sometimes has a different experience (visual or screen reader) than other browsers, and as long as the app still works, it's OK.

I'd suggest closing this bug as won't fix since despite being different from other browsers, it's not a bad or broken experience.
